src/content/blog/2024/tools-2_13.md

```diff
@@ -40,6 +41,7 @@ nf-core subworkflows install utils_nfvalidation_plugin
 4. Including the `utils_*` subworkflows in the workflow main.nf as done in [rnaseq (`dev`)](https://github.com/nf-core/rnaseq/blob/48663bffadb900e1ae4e11fb3391134cbf12ffc7/workflows/rnaseq/main.nf#L25-L30).
 5. Delete the `lib/` directory after you have confirmed everything has been ported across.
 6. Once you have merged this to `dev` the template sync PR will be updated and tell you whether you have missed anything.
+7. Be sure to check the `modules.json` file as previously installed subworkflows might have dissapeared from it.
```

Comment:
Can you be more concrete, what to check? and why would some previously installed files not be in modules.json anymore?

Reply:
added some details in the following lines